### PR TITLE
Retrieve timestamps (ETag) of plural endpoints

### PR DESCRIPTION
**Pull request by leplatrem**
Follow-up of #351

### CODE REVIEW

**Comment by leplatrem** [comment]
Tests fail because of #424

File: README.md [hunk 1]
```diff
@@ -1168,10 +1191,31 @@ Sample result:
 42
 ```
 
-#### Options
+#### options
```
Comment (glasserc) [inline README.md]: I think this should remain capitalized?

File: README.md [hunk 2]
```diff
@@ -1142,7 +1165,7 @@ Sample result:
 
 The result object exposes the following properties:
 
-- `last_modified`: the [collection's timestamp](http://kinto.readthedocs.io/en/stable/api/1.x/timestamps.html). This value is opaque and should be reused as is, eg. passing it as a `since` option (see [Generic bucket and collection options](#generic-bucket-and-collection-options))
+- `last_modified`: the [collection's timestamp](http://kinto.readthedocs.io/en/stable/api/1.x/timestamps.html).
```
Comment (glasserc) [inline README.md]: Maybe we should note that it is not the same as the timestamps you get from `getCollectionsTimestamp()` etc.

File: README.md [hunk 3]
```diff
@@ -1168,10 +1191,31 @@ Sample result:
 42
 ```
 
-#### Options
+#### options
 
-- `headers`: Custom headers object to send along the HTTP request
-- `retry`: Number of retries when request fails (default: 0)
+- `headers`: custom headers object to send along the http request
```
Comment (glasserc) [inline README.md]: I think HTTP could remain capitalized.

File: README.md [hunk 4]
```diff
@@ -552,6 +555,26 @@ Sample result:
 
 This method accepts the [generic parameters for sorting, filtering and paginating results](#generic-options-for-list-operations).
 
+### Collections list timestamp
+
+The timestmap of the collections list is used for the `since` option in the [generic parameters for sorting, filtering and paginating results](#generic-options-for-list-operations).
```
Comment (glasserc) [inline README.md]: You have `timestamp` typoed as `timestmap` a few places.

File: src/bucket.js [hunk 5]
```diff
@@ -93,6 +93,52 @@ export default class Bucket {
...
+    };
+    const { headers } = await this.client.execute(request, {
+      raw: true,
+      retry: this._getRetry(options),
+    });
+    return headers.get("ETag");
+  }
+
+  /**
+   * Retrieves the ETag of the collection list, for use with the `since` filtering option.
```
Comment (glasserc) [inline src/bucket.js]: `of the group list`?

File: README.md (hunk 2 above)
Comment (leplatrem) [reply README.md]: Actually, it is the same :) I'll add a note

**Pull request merged**